src/main/kotlin/com/empowerops/babel/AntlrExtensions.kt

```diff
@@ -7,16 +7,19 @@ import org.antlr.v4.runtime.tree.TerminalNode
 
 fun ExprContext.callsBinaryOp(): Boolean
         = childCount == 3 && getChild(0) is ExprContext
-        && (getChild(2) is ExprContext || superscript() != null)
+        && (getChild(2) is ExprContext)
```

Comment:
ah, using `getChild()` when i have more concise kotlin extensions is dumb. Will try to make more elegant.